src/features/game/expansion/components/resources/Stone.tsx

```diff
@@ -180,42 +186,44 @@ export const Stone: React.FC<Props> = ({ rockIndex, expansionIndex }) => {
   const timeLeft = getTimeLeft(rock.stone.minedAt, STONE_RECOVERY_TIME);
 
   return (
-    <div className="relative z-10 group" ref={containerRef}>
+    <div className="relative z-10 group w-full h-full" ref={containerRef}>
       <img
         src={hitbox}
         style={{
           width: `${22 * PIXEL_SCALE}px`,
           opacity: 0.3,
+          left: "50%",
+          top: "50%",
+          transform: "translate(-50%, -50%)",
         }}
-        className="z-10  cursor-pointer"
+        className="absolute z-10 cursor-pointer"
         onMouseEnter={handleHover}
         onMouseLeave={handleMouseLeave}
         onClick={strike}
       />
```

Comment:
This centers the mined stone image in the middle of the 2x1 grid of a stone.

![mined](https://user-images.githubusercontent.com/41215134/184061463-ffa8d46f-f34b-44f2-aa62-779edd853663.png)